src/index.scss

```diff
@@ -1,6 +1,48 @@
+$base-font-color: #616161;
+$primary-font-color: rgba(0, 0, 0, 0.5);
+$primary-spacing: 5px;
+
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif; }
+  font-family: Merriweather;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 15px;
+  letter-spacing: 1.4px;
+  line-height: 2em;
+  color: $base-font-color;
+}
+
+h1, h2, h3 {
+  text-rendering: optimizeLegibility;
+  margin: 1em 0 .5em;
+}
+
+.background-image {
+  background-size: cover;
+  transform: scale(1.1);
+  filter: opacity(65%) blur(5px);
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: -1;
+  background-image: url(https://static.pexels.com/photos/355770/pexels-photo-355770.jpeg);
```

Comment:
Sort your properties by type, like here: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/